src/frontend/komponenter/Oppgavebenk/OppgaveFiltrering.tsx

```diff
@@ -52,6 +52,9 @@ const OppgaveFiltering: React.FC<IOppgaveFiltrering> = ({ hentOppgaver }) => {
         : ({} as IOppgaveRequest);
 
     const [oppgaveRequest, setOppgaveRequest] = useState<IOppgaveRequest>(initOppgaveRequest);
+    const [requestFraLocalStorage, settRequestFraLocalStorage] = useState<IOppgaveRequest>(
```

Comment:
Tror det kan være lurt å fjerne #51 herfra - den gjør nok ikke så mye nytte for seg, egentlig...